portfolio/src/main/webapp/index.html

```diff
@@ -159,9 +160,20 @@ <h2>Kayla What Are You Listening to Right Now?</h2>
           </li>
         </ul>
 
+        <h2>Take a Tour of Some Important Places in My Life!</h2>
+        <div id="map"></div>
+
+        <h3> Click to see...</h3>
+        <div class="map-buttons">
+        <button id = 'cali' onclick = "changeLocation(37.4104,-122.0598)" class="button"> Where I did my CS Summer program</button>
```

Comment:
optional: the preferred way to write this is in the JS. Something like this:

```js
document.getElementById('cali').addEventListener('onclick', () => changeLocation(...) );
```

Reply:
Fixed